### PR TITLE
dynamic tasks on calender

### PR DESCRIPTION
**Pull request by nuslarry**
dynamic tasks on calender

### CODE REVIEW

File: src/main/java/seedu/task/ui/CommandBox.java [hunk 1]
```diff
@@ -36,18 +40,24 @@ private void addToPlaceholder(AnchorPane placeHolderPane) {
...
+    private void updateCalender(MainWindow mainwindow , ObservableList<ReadOnlyTask> taskList) {
+        mainwindow.updateCalender(taskList);
+    }
     @FXML
     private void handleCommandInputChanged() {
         try {
             CommandResult commandResult = logic.execute(commandTextField.getText());
-
+            //@@author A0163935X
+            System.out.println(logic.getFilteredPersonList());
```
Comment (tylerrocha) [inline src/main/java/seedu/task/ui/CommandBox.java:51]: Is this print statement leftover from debugging?

File: src/test/java/guitests/ClearCommandTest.java [hunk 2]
```diff
@@ -6,26 +7,26 @@
...
-        // //verify other commands can work after a clear command
-        // commandBox.runCommand(td.hoon.getAddCommand());
-        // assertTrue(personListPanel.isListMatching(td.hoon));
-        // commandBox.runCommand("delete 1");
-        // assertListSize(0);
-        //
-        // //verify clear command works when the list is empty
-        // assertClearCommandSuccess();
-    }
+		// //verify other commands can work after a clear command
```
Comment (tylerrocha) [inline src/test/java/guitests/ClearCommandTest.java]: Do we know why these tests are breaking?

**Review by tylerrocha — COMMENTED** [review]
Looks good. Maybe just a couple minor changes and then it's good to go

**Comment by jay500s** [comment]
the codacy test is showing there are a few unused variables in your code. Maybe remove them or double check in case you meant to use them.

**Review by whyCaiJi — APPROVED** [review]
Reviewed. Good work. Keep in mind the naming convention.

**Comment by nuslarry** [comment]
merge to master

**Review by jay500s — COMMENTED** [review]
The codacy test is showing there are a few unused variables in your code. Maybe remove them or double check in case you meant to use them.

**Pull request merged**